samples/node/basic_discovery/package.json

```diff
@@ -11,15 +11,14 @@
     "main": "./dist/index.js",
     "scripts": {
         "tsc": "tsc",
-        "prepare": "cd ../../../ && npm install && cd samples/node/basic_discovery && npm run tsc"
+        "prepare": "npm run tsc"
     },
     "devDependencies": {
         "@types/node": "^10.12.18",
         "typescript": "^3.2.2"
     },
     "dependencies": {
-        "aws-iot-device-sdk": "file:../../../",
-        "aws-crt": "0.3.0",
+        "aws-iot-device-sdk-v2": "^1.0.0",
```

Comment:
This will make local testing not work, we should probably put a comment or something about using the file:// version?

Reply:
yeah, did and undid. Javascript is crap and you can't have both, and local testing works fine anyways.

you run npm install on the parent and then on this. They're identical